test/runtests.jl

```diff
@@ -1,6 +1,7 @@
 using Showoff
 using Test
 using Dates
+using Printf
```

Comment:
Add printf as we should match the behavior to `@sprintf` as it was originally intended